src/algopy_testing/models/box.py

```diff
@@ -2,6 +2,7 @@
 
 from typing import TYPE_CHECKING
 
+from algopy_testing.constants import MAX_BOX_SIZE
```

Comment:
@boblat I think it might be a good option to move this low level box class to op.py OR rename it to an aliased class so that it doesn't clash with the new higher level interface because now we got:
1. 2 classes named Box (and i think its like that on stubs as well) 
2. 2 files named box.py 

If we move this to box.py in models but one of the class names would need to be changes to not clash (but then when exported we can alias it back)
Or we can move this lower level box interface to the op.py (so its mirroring how its done on pyi.stubs) but then the high level box stub implementation is moved here. 

What do you think? cc @daniel-makerx

Reply:
I've moved the lower level `op.Box` to the `op.py` file and the higher level `Box` interface here in this commit. We can revert it if it does not look right. 
[move low level op.Box into op.py for consistency](https://github.com/algorandfoundation/algorand-python-testing/pull/3/commits/552553e5763abe6b284bcd22a19ea3acade3326c)